testsuite/classlibrary/TestSimpleNumber.sc

```diff
@@ -54,6 +54,12 @@ TestSimpleNumber : UnitTest {
 		this.assert(actual == expected, "expected %, got %".format(expected, actual));
 	}
 
+	test_asTimeString_millisecondPrecisionRounding {
+		var expected = "00:00:01.000";
+		var actual = 0.9999.asTimeString(precision: 0.001);
+		this.assert(actual == expected, "expected %, got %".format(expected, actual));
```

Comment:
this should be `assertEquals` and have a more descriptive comment. (feel free to refactor the surrounding code to use `assertEquals` also)

Reply:
Sure.
- should I leave comments pretty much as they are?
- should I set `report` parameter of `assertEquals` to `true` (default)?

Reply:
yeah leave it at default